docs/cookbook/recipe_creating_a_locale_switcher.rst

```diff
@@ -0,0 +1,161 @@
+Creating a locale switcher
+==========================
+
+This is a full working example of creating a locale switcher.
+
+.. note::
+    This article assumes you are using Symfony 4, Flex, and autowiring.
+
+Prerequisites
+-------------
+
+Install following Sonata dependencies:
+
+* `Sonata Easy-Extends Bundle`_
+* `Sonata Intl Bundle`_
+
+Create ``LocaleSubscriber`` and ``UserLocaleSubscriber`` as explained `here`_.
+
+.. note::
+    Sonata Intl Bundle is optional, but will allow list of locales to be shown as list of languages.
```

Comment:
```suggestion
    The ``SonataIntlBundle`` bundle is optional, but will allow lists of locales to be shown as lists of languages.
```

Reply:
Not sure, it's only one list of languages...

Reply:
Probably because I again forgot "the" and "a", I am too lazy, I think it should be:

> but will allow **the** list of locales to be shown as **a** list of languages.

I will commit your changes and correct like that.